kubetest/kind/kind.go

```diff
@@ -275,6 +275,11 @@ func (d *Deployer) Build() error {
 			acceptable location until this is fixed upstream
 			https://github.com/kubernetes/kubernetes/issues/68306
```

Comment:
this seems to be resolved. should i remove:
```
exec.Command("cp", "-r", "bazel-bin/test/e2e/e2e.test", "./_output/bin/")
```
instead?

Reply:
Yeah let's try just removing that. We should also do this in the kind script.

Reply:
done